docs/conf.py

```diff
@@ -47,7 +47,8 @@ def setup(sphinx):
 extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.imgmath',
               'sphinx.ext.inheritance_diagram', 'sphinx.ext.viewcode',
               #'sphinx.ext.autosummary',
-              'numpydoc', 'cmdhelp',
+              #'numpydoc',
```

Comment:
The removal of `numpydoc` appears to be causing this error in the docs build: https://travis-ci.org/xonsh/xonsh/jobs/264786037#L1143